### PR TITLE
docs: replace apper with discover

### PR DESCRIPTION
**Pull request by amifelipek**

### CODE REVIEW

File: docs/html/pk-using.html [hunk 1]
```diff
@@ -70,9 +70,8 @@ <h2>Using graphical tools:</h2>
 <code>gnome-software</code> and <code>gnome-packagekit</code> provide a rich
 set of GTK tools for automatically updating your computer and installing software.
 </p>
-<a href="http://kde-apps.org/content/show.php/Apper?content=84745">
-Apper</a> is the name of the KDE graphical tools designed
-for PackageKit.
+<a href="https://apps.kde.org/discover/">
+Discover</a> is the name of the KDE interface for PackageKit.
```
Comment (aleixpol) [inline docs/html/pk-using.html:74]: KDE Plasma interface?

**Review by aleixpol — COMMENTED** [review]
+1 otherwise

File: docs/html/pk-using.html (hunk 1 above)
Comment (FHEK789) [reply docs/html/pk-using.html:74]: I used the same terminology as https://github.com/PackageKit/PackageKit/pull/515

**Comment by aleixpol** [comment]
Fair enough.

I see @ximion has thoughts on the topics so I'd wait for his feedback.

**Pull request closed without merging**